examples/testing/swiss-knife/src/main/scala/swissknife/SparkRandomGenDataIngress.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package swissknife
+
+import java.sql.Timestamp
+
+import cloudflow.streamlets.{ IntegerConfigParameter, StreamletShape }
+import cloudflow.streamlets.avro._
+import cloudflow.spark.{ SparkStreamlet, SparkStreamletLogic }
+import org.apache.spark.sql.Dataset
+import org.apache.spark.sql.streaming.OutputMode
+import org.apache.spark.sql.functions._
+
+import cloudflow.spark.sql.SQLImplicits._
+
+case class Rate(timestamp: Timestamp, value: Long)
+
+class SparkRandomGenDataIngress extends SparkStreamlet {
```

Comment:
Should we not try to generate deterministically?

Reply:
I would indeed prefer deterministic behavior because it is much easier to verify the actual output with the expected output

Reply:
It produces a counter now.